relay-sampling/src/lib.rs

```diff
@@ -745,23 +749,52 @@ pub struct DynamicSamplingContext {
     /// Set on transaction start, or via `scope.transaction`.
     #[serde(default)]
     pub transaction: Option<String>,
+    /// The same rate with which this trace was sampled. This is a float between 0 and 1.
```

Comment:
Please add a `#[serde(flatten)] other: BTreeMap<String, String>` at the bottom to catch and forward unknown fields.